### PR TITLE
Update metadata/Add x86-installer: MartonSalomvary.SoundCleod 2.0.0

### PR DESCRIPTION
**Pull request by quhxl**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/21838)

### CODE REVIEW

File: manifests/m/MartonSalomvary/SoundCleod/2.0.0/MartonSalomvary.SoundCleod.installer.yaml [hunk 1]
```diff
@@ -1,13 +1,26 @@
...
+  Scope: user
   InstallerUrl: https://github.com/salomvary/soundcleod/releases/download/v2.0.0/soundcleod-Setup-2.0.0.exe
   InstallerSha256: 69B0C53E2DEF0A014BBE8AA4DD91B76D926B60F95EFC3CB5B34BFAF62DB6DC8A
+  UpgradeBehavior: install
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: exe
   Scope: user
+  InstallerUrl: https://github.com/salomvary/soundcleod/releases/download/v2.0.0/soundcleod-Setup-2.0.0-ia32.exe
+  InstallerSha256: 97CC985E922DA8218D4BB1BE7046F4FEC7972339FFC28E0E7A3421C00E902C48
```
Comment (OfficialEsco) [inline manifests/m/MartonSalomvary/SoundCleod/2.0.0/MartonSalomvary.SoundCleod.installer.yaml:22]:
```suggestion
  InstallerSha256: 69B0C53E2DEF0A014BBE8AA4DD91B76D926B60F95EFC3CB5B34BFAF62DB6DC8A
  InstallerSwitches:
    Silent: --silent
  UpgradeBehavior: install
- InstallerLocale: en-US
  Architecture: x86
  InstallerType: exe
  Scope: user
  InstallerUrl: https://github.com/salomvary/soundcleod/releases/download/v2.0.0/soundcleod-Setup-2.0.0-ia32.exe
  InstallerSha256: 97CC985E922DA8218D4BB1BE7046F4FEC7972339FFC28E0E7A3421C00E902C48
  InstallerSwitches:
    Silent: --silent
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Oooo i recognize this installer screen, whenever you see this kind of installer the silent switch is `--silent`, not sure if its InstallShield, but when we get support for that we need change the InstallerType 
https://github.com/microsoft/winget-cli/issues/1247

**Comment by quhxl** [comment]
Okay... Never saw this before. USSF didn't detect it either...

**Review by OfficialEsco — APPROVED** [review]
Well tbh USSF seems to only detect Nullsoft or Inno, wingetcreate is a lot better
(no support for InstallShield either)

**Pull request merged**